peerscout/keyword_extract/keyword_extract.py

```diff
@@ -79,16 +79,16 @@ def iter_extract_keywords(
 def get_keyword_extractor(
         keyword_extract_config: KeywordExtractConfig) -> KeywordExtractor:
 
-    if not keyword_extract_config.spacy_language_model:
-        return SimpleKeywordExtractor()
-    else:
+    extractor = SimpleKeywordExtractor()
```

Comment:
I preferred the previous version. Although the else block wasn't necessary. Is your motivation to have a single return statement?

Reply:
pylint "unnecessary if else" error is what prompted me to change it

Reply:
I think it encourages you to not use `else` if the `if` block is already returning. The motivation for that I think is to reduce nested blocks.